Library/Formula/libgdiplus.rb

```diff
@@ -0,0 +1,32 @@
+class Libgdiplus < Formula
+  desc "C-based implementation of the GDI+ API for Mono"
+  homepage "http://www.mono-project.com/docs/gui/libgdiplus/"
+  url "https://github.com/mono/libgdiplus/tarball/79ae6cdb90163ba5eb601f8691ea9b7558d25371"
+  head "https://github.com/mono/libgdiplus.git", :using => :git, :tag => "master"
+  sha256 "0b84f8dd36e4e6df0877d66c816e7ef19892b52a552b1eed18e039bfb52d4de1"
+  version "3.12" # latest tag = 3.12, plus two patches.
+
+  # it depends on mono to link to System.Drawing
+  depends_on "mono"
+  depends_on "libexif"
+  depends_on "glib"
+  depends_on "cairo"
+  depends_on :x11 => :build
```

Comment:
Since this is a core part of mono, it should preferably be as easy to install as mono is. Since adding taps (or even finding them) isn't as easy, that probably excludes x11. On the other hand, even if it's slightly harder to install, unless you're doing the work for the patching (or someone else around here who wants to do it?), then it may be worth the extra hardship to install to add it to a x11 tap to get it up there. The rest can be solved with docs.